allensdk/brain_observatory/behavior/metadata/util.py

```diff
@@ -17,3 +17,23 @@ def parse_cre_line(full_genotype: str) -> Optional[str]:
     if ';' not in full_genotype:
         return None
     return full_genotype.split(';')[0].replace('/wt', '')
+
+
+def parse_age_in_days(age: str):
+    """Converts the age string into a numeric days representation
+
+    Parameters
+    ----------
+    age
+        String representation of age (ie P123)
+    """
+    if len(age) == 0:
+        return None
+
+    if age[0] != 'P':
+        return None
+    try:
+        age = int(age[1:])
+        return age
+    except ValueError:
+        return None
```

Comment:
I wonder if a warning should be given if `age` does not start with `P` or can't be coerced to an `int`?

Reply:
A warning is given by session. Please check the logic in BehaviorMetadata class.